django/db/migrations/graph.py

```diff
@@ -60,9 +119,11 @@ def forwards_plan(self, node):
         This is the list you would follow if applying the migrations to
         a database.
         """
-        if node not in self.nodes:
+        if node not in self.node_map:
             raise NodeNotFoundError("Node %r not a valid node" % (node, ), node)
-        return self.dfs(node, lambda x: self.dependencies.get(x, set()))
+        self.ensure_not_cyclic(node, lambda x: (parent.key for parent in self.node_map[x].parents))
```

Comment:
When you keep sets for `parent_keys` and `children_keys` on a `Node` you don't need a lambda here. Not sure if that's worth it though.